src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -218,6 +220,20 @@ public void bookRoom(Booking booking) {
         bookingList.add(booking);
     }
 
+
+    // to update accordingly when implementing billing system.
+    @Override
+    public void fetchBillList(Person person) {
+        String[] dummy = new String[] {"bnfajdnnlasdf"};
+        updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(dummy)));
+    }
+
+    @Override
+    public void fetchBill(Person person, String roomNum) {
+        String[] dummy = new String[]{"bnfajdnnlasdf"};
+        updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(dummy)));
+    }
```

Comment:
What does dummy here do?

Reply:
It produces an empty view for guests without bills (for now), will need to change these two methods once the bill system is implemented